src/Microsoft.DotNet.Compiler.Common/AssemblyInfoFileGenerator.cs

```diff
@@ -79,6 +79,11 @@ public static string GenerateFSharp(AssemblyInfoOptions metadata)
                 attributes[typeof(TargetFrameworkAttribute)] = EscapeCharacters(metadata.TargetFramework);
             };
 
+            if (metadata.IsServiceable)
+            {
+                attributes[typeof(AssemblyMetadataAttribute)] = "Serviceable\", \"True";
```

Comment:
Do we actually need this?  Why not just check in an AssemblyInfo with this assembly attribute set?